doc/source/operations/secret-rotation.rst

```diff
@@ -129,7 +129,7 @@ Full method
       ^redis_master_password
       ^memcache_secret_key
       _ssh_key
-         
+
```

Comment:
Are you sure this whitespace isn't required?

Reply:
This was unintended, my VScode did it automatically I think.
However, I compared with current doc and copying the block of this code gives same result between current version and this change.
If we want a precise code block, it might be better to provide a link to raw content of github gist.
Which can be downloaded by using cURL or wget